core/src/lib.rs

```diff
@@ -81,6 +81,17 @@ fn test_typed_value() {
     assert!(TypedValue::String("foo".to_string()).is_congruent_with(None));
 }
 
+/// Bit flags used in `flags0` column in temporary tables created during search,
+/// such as the `search_results`, `inexact_searches` and `exact_searches` tables.
+/// When moving to a more concrete table, such as `datoms`, they are expanded out
+/// via these flags and put into their own column rather than a bit field.
+pub enum AttributeBitFlags {
+    IndexAVET     = 1 << 0,
+    IndexVAET     = 1 << 1,
+    IndexFullText = 1 << 2,
```

Comment:
nit: I think we're considering "fulltext" one word, but I'm not militant about this.